INSTALL.md

```diff
@@ -100,67 +106,50 @@ for the relevant Linux distribution.  Alternatively, if you desire to install us
 Linux package management software such as [yum] or [apt-get], please submit a feature
 request via our [Issues] page.
 
-<a name="buildingfromsource">
-## Building from source ##
-</a>
+Advanced Installation from Source
+--------------------
 
-<a name="prerequisits">
 ### Prerequisites: ###
-</a>
-
-For broad coverage of CAF features, ease of installation, and ease of use, first
-install the following:
-
-* An OpenCoarrays-aware Fortran compiler: currently [GCC] 5.1 or later,
-* The Fortran compiler's companion C compiler that supports the C99 standard, and
-* An MPI implementation that supports MPI 3.0 and is built by the aforementioned
-  Fortran and C compilers (preferably the [MPICH] or [MVAPICH] implementations for
-  robustness and high performance)
-
-The [install_prerequisites] directory contains experimental [buildgcc], [buildmpich], and
-[buildcmake] bash shell scripts that can download and build any one of several versions of the
-[GCC]  C, C++, and Fortran compilers; the [CMake] build sytem; and the [MPICH] communication
-library.  (Because newer parts of the GNU Fortran compiler gfortran are written in C++,
-installing the GNU Fortran compiler from source requires also installing the GNU C++ compiler
-g++.)  The CMake scripts that build OpenCoarrays also copy [buildgcc], [buildmpich], and
-[buildcmake] into the "bin" directory of the OpenCoarrays installation for later use.
-
-We have tested the build scripts  on OS X and Linux. Please submit suggestions for improving
-the scripts to our [Issues] page or preferably suggeste edits by forking a copy of the
-[OpenCoarrays] repository, making the suggested edits, and submitting a pull request.
-
-If installing the above prerequisites is infeasible, then a limited coverage of CAF
-features is available via the OpenCoarrays "caf" compiler wrapper and the
-[opencoarrays] module, for which the installation prerequisites are the following:
-
-* A Fortran compiler that supports the C-interoperability features of Fortran 2003,
-* The Fortran compiler's companion C compiler that supports the C99 standard, and
-* An MPI implementation that supports MPI 3.0 and is built by the aforementioned
-  Fortran and C compilers (preferably the [MPICH] or [MVAPICH] implementations for
-  robustness and high performance)
-
-<a name="cmake">
+
+The prerequisites below and their dependencies are recommended for the broadest coverage of CAF features. If a prerequisite is missing or outdated, the [install.sh] script will prompt the user for permission to download, compile, and install it. Before doing so, [install.sh] will verify whether that prerequisite's prerequisites are present and will recursively traverse the dependency tree until reaching an acceptable prerequisite or reaching the end of a branch.
+
```

Comment:
Here is the text you wanted to change. If everything looks good I'll tag it and mint a release.